features/storage/csi.feature

```diff
@@ -32,3 +32,40 @@ Feature: CSI testing related feature
     Given admin uses the "csihostpath" project
     Given the pod named "my-csi-app" status becomes :running
 
+  # @author chaoyang@redhat.com
+  @admin
+  Scenario Outline: Configure 'Retain' reclaim policy
```

Comment:
I think with cuke5 we can use `Scenario` instead of `Scenario Outline`. cc @liangxia

Reply:
@jhou1  If the `Examples` part contains case ID for each line, `Scenario Outline` is fine.

Reply:
Thanks!